includes/onnxruntime/onnxruntime_c_api.h

```diff
@@ -49,7 +52,11 @@ extern "C" {
 #else
 #define ORT_EXPORT
 #endif
+#if defined(__MINGW32__)
```

Comment:
Do you need to support mingw32?

Reply:
Yes, some people asked that, and this small fixing can enable it.

Reply:
You may consider send a PR to the main repo.